easybuild/scripts/bootstrap_eb.py

```diff
@@ -767,6 +771,10 @@ def main():
     if sourcepath is not None:
         info("Fetching sources from %s..." % sourcepath)
 
+    forcedversion = EASYBUILD_BOOTSTRAP_FORCE_VERSION
+    if EASYBUILD_BOOTSTRAP_FORCE_VERSION is not None:
+        info("Forcing specified version %s..." % forcedversion)
```

Comment:
If forcedversion isn't use further down, then drop it and use EASYBUILD_BOOTSTRAP_FORCE_VERSION in the info(...).
If it is used then changing the if EASYBUILD.... to
if forcedversion:
makes it easier to read.

Reply:
@akesandgren Well, it *is* used further down, since it's being passed to `stage1`.

So, your second suggestion is right on the spot, please use `if forcedversion:` above